src/VisualStudio/Core/Def/Preview/PreviewUpdater.cs

```diff
@@ -15,22 +16,25 @@
 
 namespace Microsoft.VisualStudio.LanguageServices.Implementation.Preview
 {
-    internal partial class PreviewUpdater : ForegroundThreadAffinitizedObject
+    internal partial class PreviewUpdater : ForegroundThreadAffinitizedObject, IDisposable
     {
-        private PreviewDialogWorkspace? _previewWorkspace;
+        private ReferenceCountedDisposable<PreviewDialogWorkspace>? _previewWorkspace;
```

Comment:
wow... this is lazily created and assigned.... i hate this :D (same problem before your code).

Reply:
➡️ Planning to leave this one as-is since it's the same as before